sphinx/themes/basic/static/searchtools.js_t

```diff
@@ -256,7 +256,7 @@ var Search = {
             displayNextItem();
           });
         } else if (DOCUMENTATION_OPTIONS.HAS_SOURCE) {
-          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[0] + '.txt',
+          $.ajax({url: DOCUMENTATION_OPTIONS.URL_ROOT + '_sources/' + item[5] + '.txt',
```

Comment:
I always uses `python -m SimpleHTTPServer 8080` command. It invokes HTTP Server locally at 8080 port. so you can access the directory through `http://localhost:8080/`.